tika-transcribe/src/main/java/org/apache/tika/transcribe/transcribe/AmazonTranscribe.java

```diff
@@ -0,0 +1,193 @@
+    }
+
+    @Override
+    public void startTranscribeVideo(String fileName, String jobName) throws TikaException, IOException {
+        if (!isAvailable())
+            return;
+        //TODO
+
+    }
+
+    /**
+     * Audio to text function with language specification
+     * @param fileName
+     * @param sourceLanguage
+     * @return Transcribed text
+     * @throws TikaException
+     * @throws IOException
+     */
+    @Override
+    public void startTranscribeVideo(String fileName, LanguageCode sourceLanguage, String jobName) throws TikaException, IOException {
+        if (!isAvailable())
+            return;
+        //boolean validSourceLanguageFlag = validSourceLanguages.contains(sourceLanguage); // Checks if sourceLanguage in validSourceLanguages O(1) lookup time
+
+        //if (!validSourceLanguageFlag) { // Throws TikaException if the input sourceLanguage is not present in validSourceLanguages
+        //    throw new TikaException("Provided Source Language is Not Valid. Run without language parameter or please select one of: " +
+        //           "en-US, en-GB, es-US, fr-CA, fr-FR, en-AU, it-IT, de-DE, pt-BR, ja-JP, ko-KR"); }
+        //TODO
+
+    }
+
+    /**
+     * @return Valid AWS Credentials
+     */
+	public boolean isAvailable() {
+		return this.isAvailable;
+	}
+
+    /** Gets Transcriptioni result from AWS S3 bucket given bucketNamee and key
+     * @param key
+     * @return
+     */
+    @Override
+    public String getTranscriptResult(String key) {
+        TranscriptionJob transcriptionJob = retrieveObjectWhenJobCompleted(key);
+        if (transcriptionJob != null && !TranscriptionJobStatus.FAILED.equals(transcriptionJob.getTranscriptionJobStatus())) {
+            return amazonS3.getObjectAsString(this.bucketName, key + ".json");
+        } else
+            return null;
+    }
+
+    /**
+     * Private helper function to get object from s3
+     * @param key
+     * @return
+     */
+    private TranscriptionJob retrieveObjectWhenJobCompleted(String key) {
+        GetTranscriptionJobRequest getTranscriptionJobRequest = new GetTranscriptionJobRequest();
+        getTranscriptionJobRequest.setTranscriptionJobName(key);
+
+        while (true) {
+            GetTranscriptionJobResult innerResult = amazonTranscribe.getTranscriptionJob(getTranscriptionJobRequest);
+            String status = innerResult.getTranscriptionJob().getTranscriptionJobStatus();
+            if (TranscriptionJobStatus.COMPLETED.name().equals(status) ||
+                    TranscriptionJobStatus.FAILED.name().equals(status)) {
+                return innerResult.getTranscriptionJob();
+            }
+        }
+    }
+
+    /**
+     * Call this method in order to upload a file to the Amazon S3 bucket.
+     * @param bucketName
+     * @param fileName
+     * @param fullFileName
+     */
+    @Override
+    public void uploadFileToBucket(String bucketName, String fileName, String fullFileName) {
```

Comment:
This needs to exist in the AWS implementation but NOT in the Transcribe Interface.